CONTRIBUTING.md

```diff
@@ -0,0 +1,160 @@
+# Contribution guide
+
+**Want to contribute? Great!**
+We try to make it easy, and all contributions, even the smaller ones, are more than welcome.
+This includes bug reports, fixes, documentation, examples...
+But first, read this page (including the small print at the end).
+
+## Legal
+
+All original contributions to Kogito are licensed under the
+[ASL - Apache License](https://www.apache.org/licenses/LICENSE-2.0),
+version 2.0 or later, or, if another license is specified as governing the file or directory being
+modified, such other license.
+
+## Issues
+
+TrustyAI-explainability uses [JIRA to manage and report issues](https://issues.redhat.com/projects/FAI/).
+
+If you believe you found a bug, please indicate a way to reproduce it, what you are seeing and what you would expect to see. 
+Don't forget to indicate your TrustyAI-explainability, Java, Maven, Quarkus/Spring, GraalVM version.
+
+### Checking an issue is fixed in main
+
+Sometimes a bug has been fixed in the `main` branch of TrustyAI-explainability and you want to confirm it is fixed for your own application. 
+Testing the `main` branch is easy and you can build TrustyAI-explainability all by yourself.
+
+## Creating a Pull Request (PR)
+
+To contribute, use GitHub Pull Requests, from your **own** fork.
+
+- PRs should be always related to an open JIRA issue. If there is none, you should create one.
+- Try to fix only one issue per PR.
+- Make sure to create a new branch. Usually branches are named after the JIRA ticket they are addressing. E.g. for ticket "FAI-XYZ An example issue" your branch should be at least prefixed with `FAI-XYZ`. E.g.:
+
+        git checkout -b FAI-XYZ
+        # or
+        git checkout -b FAI-XYZ-my-fix
+
+- When you submit your PR, make sure to include the ticket ID, and its title; e.g., "FAI-XYZ An example issue".
+- The description of your PR should describe the code you wrote. The issue that is solved should be at least described properly in the corresponding JIRA ticket.
+- If your contribution spans across multiple repositories, use the same branch name (e.g. `FAI-XYZ`).
+- If your contribution spans across multiple repositories, make sure to list all the related PRs.
+
+### Java Coding Guidelines
+
+We decided to disallow `@author` tags in the Javadoc: they are hard to maintain, especially in a very active project, and we use the Git history to track authorship. GitHub also has [this nice page with your contributions](https://github.com/trustyai-explainability/trustyai-explainability/graphs/contributors).
+
+Copyright headers format is enforced during build time. In order to automatically format your files, you could run the following Maven command:
+```bash
+mvn com.mycila:license-maven-plugin:format
+```
+
+We follow [Kogito project's codestyle](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config), so please make sure you have configured your IDE accordingly.
```

Comment:
Same comment here, we might need to copy the formatting plugin conf (I would postpone to different ticket)